media/css/pebbles/includes/mixins/_at2x.scss

```diff
@@ -0,0 +1,56 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+// A helper mixin for applying high-resolution background images (based on http://www.retinajs.com)
+// File-names should follow:
+//  example.png
+//  example-high-res.png
+// Usage:
+//  @include at2x('/media/img/foo.png', 100px, 100px);
+
+@mixin at2x($path, $w: auto, $h: auto) {
+    $hr-suffix: '-high-res';
+
+    // Set a counter and get the length of the image path.
+    $position: -1;
+    $strpath: '#{$path}';
+    $length: str-length($strpath);
+
+    // Loop through the image path and find the position of
+    // the dot to determine the file extension.
+    @for $i from $length through $length - 10 {
```

Comment:
Yeah, 10 seems like an arbitrary number. 1 might work just as well, or 100, or 23, or 42, etc. You might get a negative integer for short filenames but that's still lower so still increments the loop. I can do some testing with different paths to see if there's a good reason for using 10 instead of 1 but I'll add a comment in the meantime.